PLAYGROUNDS.md

```diff
@@ -0,0 +1,142 @@
+# Deno Deploy Playgrounds
+
+Playgrounds provide a web-based development environment that lets you create, edit, and deploy applications directly in your browser without any local setup or Git repositories.
+
+## Getting Started
+
+### Creating Your First Playground
+
+1. Navigate to your organization's playground section: `/{org}/~/playgrounds`
+2. Click **"New Playground"**
+3. Choose from available templates:
+   - **Hello World** - Basic Deno server with HTTP handling
+   - **Hono** - Web framework for building fast APIs
+   - **Next.js** - React-based full-stack framework
+
+Your playground will be created instantly with template files ready for editing.
+
+### The Playground Interface
+
+The playground editor features a multi-panel layout:
+
+- **Left Panel**: File explorer for navigation and file management
+- **Center Panel**: Monaco code editor with syntax highlighting and IntelliSense
+- **Right Panel**: Live preview iframe and HTTP explorer
+- **Bottom Panel**: Build logs and deployment status
+
+All panels are resizable and can be collapsed/expanded as needed.
+
+## Core Features
+
+### Code Editing
+
+- **Full-featured Editor**: Monaco editor with TypeScript support, syntax highlighting, and auto-completion
+- **Multi-file Support**: Edit multiple files with tab-based navigation
+- **Auto-save**: Changes are automatically saved as you type
+- **IntelliSense**: Smart code completion and error detection
+
+### File Management
+
+- **Create Files**: Add new files and folders through the file explorer
+- **Rename/Delete**: Right-click context menu for file operations
```

Comment:
Is that correct? I don'd see a right-click menu and the deleting and renaming of files happens via the delete icon and clicking on the file title in the editor